target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+ad7200)
+	ucidef_set_led_usbdev "usb1" "USB 1" "${board}:blue:usb_1" "2-1"
+	ucidef_set_led_usbdev "usb2" "USB 2" "${board}:blue:usb_3" "4-1"
```

Comment:
@dissent1 Would you please have a look at this PR at all. It is basically a clone of the C2600 and you have done a lot of changes to the C2600 so that I got literally lost. It would be perfect if you would take the job of reviewing this PR.